ansible/server-state-playbooks/www/requirements.yml

```diff
@@ -16,9 +13,6 @@
   src: https://github.com/openmicroscopy/ansible-role-jekyll-build.git
   version: 1.2.0
 
-- name: openmicroscopy.lvm-partition
```

Comment:
Is `lvm-partition` pulled as a dependency of another role? What happens if a deployment includes a version < 1.1.0?